http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/CookieSupport.scala

```diff
@@ -47,7 +44,34 @@ object CookieSupport {
   private def buildRawUnsetCookie(): RawHeader =
     RawHeader(
       name = "Set-Cookie",
-      value = s"$tokenCookieName=unset; expires=Thu, 01 Jan 1970 00:00:00 GMT; Path=/; Secure; SameSite=None; HttpOnly"
+      value = s"$tokenCookieName=unset; Expires=Thu, 01 Jan 1970 00:00:00 GMT; Path=/; Secure; SameSite=None; HttpOnly"
```

Comment:
just confirming the case change here is intentional?

Reply:
Yeah, intentional!